core/src/main/scala/kafka/admin/DeleteRecordsCommand.scala

```diff
@@ -18,16 +18,19 @@
 package kafka.admin
 
 import java.io.PrintStream
+import java.util
 import java.util.Properties
 
-import kafka.admin.AdminClient.DeleteRecordsResult
 import kafka.common.AdminCommandFailedException
-import kafka.utils.{CoreUtils, Json, CommandLineUtils}
+import kafka.utils.{CommandLineUtils, CoreUtils, Json}
 import org.apache.kafka.common.TopicPartition
 import org.apache.kafka.common.utils.Utils
+import org.apache.kafka.clients.admin.{RecordsToDelete, AdminClient => JAdminClient}
```

Comment:
Why not import it as `AdminClient`? We don't use the other AdminClient anymore so we don't need to rename it.

Reply:
The Scala AdminClient is going away and there is no need to have this rename in classes that don't need it.

Reply:
Sure, I removed the alias. Since the old AdminClient is in the same package, the normal import doesn't seem work; hence the `admin.AdminClient` reference.